### ISSUE TITLE
Duplicate attributes for nested Sprig elements

### ISSUE THREAD

**Issue by rene-haefliger** [putyourlightson/craft-sprig#208]
### Duplicate attributes for nested Sprig elements
With nested Sprig element attributes are created twice.

### To reproduce
Steps to reproduce the behaviour:
1. Create parent file and load a child file with `{{ sprig('shop/category/_categoryProducts', {'categoryId': category.id }) }}`
2. Create a child file and load a child file with `{{ sprig('shop/category/_categoryProductAddToWishlist', {'purchasableId': variant.id}) }}`
3. Create a child file with this content: `<button data-sprig data-s-vals="{{ { elementId: purchasableId, listTypeHandle: 'wishlist'}|json_encode }}" data-s-method="post" data-s-action="wishlist/items/add" id="add-to-wishlist-{{ purchasableId }}" class="shop-button add-to-wishlist js-tooltip-trigger" data-tooltip-position="bottom"  title="Auf Merkliste setzen"></button>`

### Expected behaviour
No duplicate attributes.

### Demo
[Test-Page](https://commerce.klarplan.dev/draussen-zuhause/grill/gasgrill)

### Versions
- Sprig version: 1.12.1. 
- Craft version: 3.7.37

**Comment by bencroker** [comment]
Can you show me the output that contains the duplicate attributes?

**Comment by rene-haefliger** [comment]
```
<div id="component-tjoafk" class="sprig-component" data-hx-target="this" data-hx-include="this" data-hx-trigger="refresh" data-hx-get="https://commerce.klarplan.dev/index.php?p=actions/sprig-core/components/render" data-hx-vals="{&quot;sprig:siteId&quot;:&quot;852c3444b7df72fe30b917c58710dc3d4f8e746afe8df1a0aed4830befaee1091&quot;,&quot;sprig:template&quot;:&quot;c2cf754f7b5092ab6e547c9bc2b34be65dae579a46dabe9447ca56c05967f610shop\/category\/_categoryProductAddToWishlist&quot;,&quot;sprig:variables[purchasableId]&quot;:&quot;cbd8a9f5dcb2cb0f61fc4d2cddcffa34f09fe39bdba3f05d50764f97b511bd8b13770&quot;}">
    <button id="add-to-wishlist-13770" class="shop-button add-to-wishlist js-tooltip-trigger" title="Auf Merkliste setzen" data-sprig data-s-vals="{&quot;elementId&quot;:13770,&quot;listTypeHandle&quot;:&quot;wishlist&quot;}" data-s-method="post" data-s-action="wishlist/items/add" data-tooltip-position="bottom" data-hx-headers="{&quot;X-CSRF-Token&quot;:&quot;LaXrB7bbaHi4mQPfAFvoOTDRMOPK6G4_1olCvrDCJTFbVrhGnAyAVjPf1kBQhZX0RbF9vXYDhvfTIeb7SuoHhDXk1TcaARpEAEw1gpy2Ga8LQB_PoNyvxANU23eFe0lMAYNgNGnlOZQ8zvUjGrZSf3NZfnIJOXG5SjoKlLXtQcezxRJYLzesgp0vjGt3gwyT1JYSfuKmdOy-Jyjnq0-zchWiYV_naJv_twNNgNB1yt6PZ3AhL40P7XrRbk-94Xgx4a2WdoRM3E8w5JGxFoCfLo4_s1fT4QN-NypwYnhRu6yDfJkydCj9l-3K9DFjukHmQ3B8YOFZBljRCaEOnbJuaew3MCMGTD5lSz_zElU8zM3YHqrBk0tvW1nhwNxV9V4-8nrC7Ooj&quot;}" data-hx-post="https://commerce.klarplan.dev/index.php?p=actions/sprig-core/components/render&amp;sprig:action=7da62051b1805cba9ff083ba42d4ce618fa77e7503620f2df9bff7dd41ab91bcwishlist/items/add" data-hx-vals="{&quot;elementId&quot;:13770,&quot;listTypeHandle&quot;:&quot;wishlist&quot;}" data-hx-headers="{&quot;X-CSRF-Token&quot;:&quot;LaXrB7bbaHi4mQPfAFvoOTDRMOPK6G4_1olCvrDCJTFbVrhGnAyAVjPf1kBQhZX0RbF9vXYDhvfTIeb7SuoHhDXk1TcaARpEAEw1gpy2Ga8LQB_PoNyvxANU23eFe0lMAYNgNGnlOZQ8zvUjGrZSf3NZfnIJOXG5SjoKlLXtQcezxRJYLzesgp0vjGt3gwyT1JYSfuKmdOy-Jyjnq0-zchWiYV_naJv_twNNgNB1yt6PZ3AhL40P7XrRbk-94Xgx4a2WdoRM3E8w5JGxFoCfLo4_s1fT4QN-NypwYnhRu6yDfJkydCj9l-3K9DFjukHmQ3B8YOFZBljRCaEOnbJuaew3MCMGTD5lSz_zElU8zM3YHqrBk0tvW1nhwNxV9V4-8nrC7Ooj&quot;}" data-hx-post="https://commerce.klarplan.dev/index.php?p=actions/sprig-core/components/render&amp;sprig:action=7da62051b1805cba9ff083ba42d4ce618fa77e7503620f2df9bff7dd41ab91bcwishlist/items/add" data-hx-vals="{&quot;elementId&quot;:13770,&quot;listTypeHandle&quot;:&quot;wishlist&quot;}"></button>
</div>
```
[Here are the error messages](https://validator.w3.org/nu/?doc=https%3A%2F%2Fcommerce.klarplan.dev%2Fdraussen-zuhause%2Fgrill%2Fgasgrill)

**Comment by bencroker** [comment]
Thanks, fixed in https://github.com/putyourlightson/craft-sprig-core/commit/af814ff6f9b518b77b76a366863fa3ccf58f0999 and released in version 1.12.2.

**Comment by rene-haefliger** [comment]
Thank you very much. You are awesome.
